packages/java/src/browser/java-commands.ts

```diff
@@ -46,23 +42,33 @@ const CONTEXT_ID = 'java.editor.context';
 export class JavaEditorContext implements KeybindingContext {
     readonly id = CONTEXT_ID;
 
-    constructor(@inject(EditorManager) protected readonly editorService: EditorManager) { }
+    @inject(EditorKeybindingContext)
+    protected readonly editorContext: EditorKeybindingContext;
 
     isEnabled(arg?: Keybinding) {
-        return this.editorService && !!this.editorService.currentEditor && (this.editorService.currentEditor.editor.document.uri.endsWith(".java"));
+        return this.editorContext.isEnabled(arg) && !!this.widget;
+    }
+
+    get widget(): EditorWidget | undefined {
+        const widget = this.editorContext.widget;
```

Comment:
just asking, should it be current or active editor?

Reply:
`active` is correct. This is basically an extension of the editor context.